libs/deal-ticket/src/components/deal-ticket-manager.tsx

```diff
@@ -49,12 +48,12 @@ export const DealTicketManager = ({
   };
 
   useEffect(() => {
-    if (transaction.status !== VegaTxStatus.Default) {
+    if (transaction.status !== VegaTxStatus.Default || finalizedOrder) {
```

Comment:
I added this because the `Order placed` modal would not appear otherwise

Reply:
It would get stuck into `Awaiting network confirmation` as the tx status comes very quickly into Default. I tried to quickly place an additional order and it works fine.

Reply:
It is actually the same dialog that reopens. 
As per your suggestion, I will refactor this dialog to be used for both submission and cancellation (without any dialog wrappers).
I think I should update this in a separate PR.